Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.6.3
+
+ENV SHELL="/bin/bash"
+
+RUN apt-get update && \
+    apt-get install -y git \
+                       wget \
+                       python3-pip \
+                       vim
```

Comment:
@jtwalsh0  I included it because I was using it to alter the configuration files inside the docker. However, now I have it set up to pull the config file into the docker by copying it. Do you think that partners will potentially enter the docker to edit the files?